digital-deck-server/routes/player.js

```diff
@@ -0,0 +1,45 @@
+const router = require('express').Router();
+const {getPlayer, getSession} = require('../services/utilities');
+const cardDrawService = require('../services/cardDrawService');
+
+router.post('/playcard', async function(req, res) {
+    var sessionId = req.body.sessionId; 
+    var playerId = req.body.playerId; 
+    var cardIndex = req.body.cardIndex;  // there should only be one of this required, either the card or the index
+    var card = req.body.card;           // since the card needs to be added to the table, sending the index of the card in player's hand shouldn't be required
+
+    var session = getSession(sessionId);
+    var table = session.table;
+    var player = getPlayer(session, playerId);
+    
+    try{
+        player.removeCard(cardIndex);
+        table.addCardTop(card);
+        res.status(200).send();
+    }
```

Comment:
Move this to it's service maybe.

Reply:
also, for kinda the same reason; after db we will be updating the database when handling this request and I don't think the service layer should update anything in database